src/components/request/RequestForm.tsx

```diff
@@ -285,21 +285,44 @@ const RequestForm: FC = () => {
         <RequestFormParametersWrapper>
           <ParametersHeader>Parameters</ParametersHeader>
           <ParametersValuesWrapper>
-            <ParametersValueHeader>Proposal bond:</ParametersValueHeader>
+            <ParametersValueHeader>
+              {flags.CanPropose
+                ? "Proposal bond"
+                : flags.CanDispute || flags.InDvmVote || flags.CanSettle
+                ? "Disputer bond"
+                : ""}
+            </ParametersValueHeader>
             <ParametersValue>
               <BondLogo src={logo} alt="bond_img" /> {collateralSymbol}{" "}
               {totalBond ? prettyFormatNumber(Number(totalBond)) : ""}
             </ParametersValue>
           </ParametersValuesWrapper>
           <ParametersValuesWrapper>
-            <ParametersValueHeader>Proposal reward:</ParametersValueHeader>
+            <ParametersValueHeader>
+              {flags.CanPropose
+                ? "Proposal reward"
+                : flags.CanDispute || flags.InDvmVote || flags.CanSettle
+                ? "Disputer reward"
+                : ""}
+            </ParametersValueHeader>
             <ParametersValue>
               <BondLogo src={logo} alt="bond_img" /> {collateralSymbol}{" "}
               {reward ? prettyFormatNumber(Number(reward)) : ""}
             </ParametersValue>
           </ParametersValuesWrapper>
+          {flags.CanDispute || flags.InDvmVote || flags.CanSettle ? (
+            <ParametersValuesWrapper>
+              <ParametersValueHeader>Sponsor Reward</ParametersValueHeader>
+              <ParametersValue>
+                <BondLogo src={logo} alt="bond_img" /> {collateralSymbol}{" "}
+                {reward ? prettyFormatNumber(Number(reward)) : ""}
+              </ParametersValue>
+            </ParametersValuesWrapper>
+          ) : null}
           <ParametersValuesWrapper>
-            <ParametersValueHeader>Liveness period: </ParametersValueHeader>
+            {flags.CanDispute && (
```

Comment:
could simplify this a bit by hiding the entire `ParametersValuesWrapper` if you cant dispute, and remove the individual checks on line 323 and 327

Reply:
Fair!